setup.py

```diff
@@ -6,9 +6,9 @@
     long_description_content_type="text/markdown",
     install_requires=[
         "dbus-python==1.2.16",
-        "Flask==1.1.2",
-        "Flask-SocketIO==5.0.1",
-        "eventlet==0.31.0",
+        "Flask~=2.0.0",
```

Comment:
Should use the latest Flask version, program seems to break when using 2.0.0.
https://github.com/Brikwerk/nxbt/pull/67#issuecomment-1229361217

Reply:
I assume the referenced comment is still on the original 1.1.2 dependency. i still cannot confirm that this change does not work. also, its technically using 2.0.3. however since flask moved on we can try bumping the version again. i'm just not sure if this will get merged at some point.
@Brikwerk are you interested in these changes? are there any community guidelines? and would you be willing to publish an updated package on pypi, as it seems there is still plenty of interrest in your project.